### ISSUE TITLE
place=square object described as bicycle parking

### ISSUE THREAD

**Issue by aceman444** [osmandapp/OsmAnd#10061]
When selecting the object https://www.openstreetmap.org/relation/7836129, a multipolygon of a place=square, it is properly outlined. However The details coming from the bottom of the screen is for a completely different object https://www.openstreetmap.org/node/4122963038 (a bicycle parking). The 2 objects just have a same name, but are otherwise unrelated.

![bug-OsmAnd-place-bicycle-small](https://user-images.githubusercontent.com/6699462/96798653-9eff8200-1401-11eb-8c6a-5c44306eb1bf.png)

### Your Environment
OsmAnd Version: 3.8.3
Android/iOS version: 10
Offline map of Slovakia

**Comment by vshcherb** [comment]
Looks like We need to add place=square to POI

**Comment by xmd5a2** [comment]
@vshcherb we already have place=square as POI
poi_types:
```xml
<poi_type name="square" tag="square" value="yes" name_only="true" no_edit="true"/>
```
rendering_types:
```xml
<entity_convert pattern="tag_transform" from_tag="place" from_value="square" to_tag1="square" to_value1="yes" map="no" routing="no"/>
```

**Comment by aceman444** [comment]
Even if there was no recognition of place=square as a POI (which there is as it is rendered with a large font), how does it get mixed with a different object?

The bicycle parking isn't even inside the area of the place polygon, it can be seen to the left of it behind the tram platform (Nam. L. Stura).

**Comment by Dima-1** [comment]
![Screenshot_1617111554](https://user-images.githubusercontent.com/31807095/113001073-1c7daa00-9179-11eb-9a4a-81747355369d.png)
Please update application, issue was fixed

**Comment by aceman444** [comment]
Thank you, I will check it out. I have seen it at different places, where multiple objects close by had the same name.

**Comment by aceman444** [comment]
I can't see the problem anymore even without upgrade. Is it possible this got fixed on 3.9.10 too? Is it solved in the .obf map file? You could have linked to the pull request or patch that fixed it. Thanks.

**Comment by aceman444** [comment]
Ok, it may also be accidentally fixed by the fact the bicycle_parking on the square got renamed to a different name so OsmAnd 3.9 is no longer confusing the objects.
